cypress/e2e/filter/filterCases.cy.ts

```diff
@@ -431,6 +431,83 @@ describe("Case list", () => {
       cy.get("tr").not(":first").should("have.length", 8)
     })
 
+    it("Should display cases filtered for a custom date range", () => {
+      const force = "011111"
+
+      cy.task("insertCourtCasesWithFields", [
+        { courtDate: new Date("2023-01-1"), orgForPoliceFilter: force },
+        { courtDate: new Date("2023-02-1"), orgForPoliceFilter: force },
+        { courtDate: new Date("2022-12-1"), orgForPoliceFilter: force },
+        { courtDate: new Date("2022-11-15"), orgForPoliceFilter: force },
+        { courtDate: new Date("2022-11-2"), orgForPoliceFilter: force },
+        { courtDate: new Date("2022-10-30"), orgForPoliceFilter: force },
+        { courtDate: new Date("2022-10-15"), orgForPoliceFilter: force },
+        { courtDate: new Date("2022-10-1"), orgForPoliceFilter: force },
+        { courtDate: new Date("2022-09-15"), orgForPoliceFilter: force },
+        { courtDate: new Date("2021-12-15"), orgForPoliceFilter: force },
+        { courtDate: new Date("2021-02-10"), orgForPoliceFilter: force },
+        { courtDate: new Date("2020-05-30"), orgForPoliceFilter: force },
+        { courtDate: new Date("2019-05-10"), orgForPoliceFilter: force }
+      ])
+
+      cy.visit("/bichard")
+
+      // Tests for all cases in 2022
+      cy.get("button#filter-button").click()
```

Comment:
We could create a reusable function from the following lines:
```
cy.get("button#filter-button").click()
cy.get("#custom-date-range").click()
cy.get("#date-from").type("1999-01-01")
cy.get("#date-to").type("2000-12-31")
```

Reply:
@C-gyorfi  @monotypical  thoughts on this?